src/main/cpp/chassis/HolonomicDrive.cpp

```diff
@@ -151,6 +153,17 @@ void HolonomicDrive::Run()
             m_moveInfo.headingOption = ChassisOptionEnums::HeadingOption::MAINTAIN;
         }
 
+        // if we're in climb mode, make robot drive robot oriented
```

Comment:
I don't think this change should be here; rather I think it should be in InitChassisMovement

Reply:
Would that not get overridden from the above m_moveInfo set to Field Drive?

Reply:
Lets make sure to update this branch from main and lets talk through this and try ti tomorrow. So Nick can climb easier

Reply:
The problem is that we might just as well skip all of the logic above here because if we are not climb mode, we switch to field oriented drive.  What about if we picked one of the other modes previously (say trajectory), you just set it to field oriented.  If you don't want to set it in the init methods, then we should at least do 

if (climbmode)
{
}
else if (isAlignGamePieceSelected)

at like line 87

Reply:
Closing pull request, we have @witcpalek branch and we have the robot drive button in the comp branch